programs/drift/src/math/margin.rs

```diff
@@ -236,6 +238,51 @@ pub fn calculate_user_safest_position_tiers(
     Ok((safest_tier_spot_liablity, safest_tier_perp_liablity))
 }
 
+pub fn calculate_perp_fuel_bonus(
+    perp_market: &PerpMarket,
+    base_asset_value: i128,
+    fuel_bonus_numerator: i64, // fuel_perp_delta: Option<(u16, i64)>
+) -> DriftResult<u64> {
+    let result: u64 = if base_asset_value.unsigned_abs() < 1 {
+        0_u64
+    } else {
+        base_asset_value
+            .unsigned_abs()
+            .safe_mul(fuel_bonus_numerator.cast()?)?
+            .safe_mul(perp_market.fuel_boost_funding.cast()?)?
+            .safe_div(FUEL_WINDOW_U128)?
```

Comment:
why do you divide by this? how does a user interacting w contract more frequently affect this?